server/tests-py/graphql_server.py

```diff
@@ -53,6 +55,8 @@ def post(self, request):
 class User(graphene.ObjectType):
     id = graphene.Int()
     username = graphene.String()
+    blah = graphene.String()
```

Comment:
Perhaps the role of this field `blah` can be made more clear by renaming it into something like `generate_error`, so that future readers know why we'd explicitly add a field that cannot be queried.

Reply:
Yes, I'll rename the field to `generateError`.